src/Agent/NewRelic/Agent/Core/Configuration/DefaultConfiguration.cs

```diff
@@ -1691,7 +1695,16 @@ public virtual bool TransactionEventsTransactionsEnabled
         #region Transaction Tracer
 
         public virtual TimeSpan TransactionTraceApdexF { get { return TransactionTraceApdexT.Multiply(4); } }
-        public virtual TimeSpan TransactionTraceApdexT { get { return TimeSpan.FromSeconds(ServerOverrides(_serverConfiguration.ApdexT, 0.5)); } }
+        public virtual TimeSpan TransactionTraceApdexT
+        {
+            get
+            {
+                if (ServerlessModeEnabled) // get apdex_t from environment variable if running in serverless mode
```

Comment:
I think we can allow the environment variable override even if we aren't in serverless mode, right?

Reply:
It normally comes from the connect response, so I don't know if we want to allow an override locally or not, especially without having a proper config setting for it.

Reply:
We have some other server side config option overrides, but we can revisit when we do our eventual "overrides for everything" work.